config-rules.tf

```diff
@@ -5,7 +5,8 @@ module "aws_config_rules_us_east_1" {
   source_recorder               = module.aws_config_recorder_us_east_1.aws_config_configuration_recorder_id
   required_tags_enabled         = true
   required_tags = {
-    tag1Key = "Environment" # Yes, the actual required format is tag#Key and the required key
+    tag1Key   = "AWS_Backup" # Yes, the actual required format is tag#Key and the required key
```

Comment:
@jason-drouhard - What is the backup tag that was used in other environments? I don't think it would have had an underscore in it, and I think I would prefer to be consistent if that's the case.

Reply:
To my knowledge, we didn't have a backup defined in either the account or infrastructure starters.. I can double check and make sure that AWS_Backup is the key I used in the infrastructure starter, but if you have a preference for what the key should be, just let me know. AWS_Backup just made sense to me at the time.

Reply:
I think we have something we used on Property Masters and Tag. I'm not necessarily opposed to it, but it would be good for them all to match and be tracked by the same config rules.